src/components/footer/Footer.js

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components'
+
+function RenderFooter() {
+  return (
+    <Footer>
```

Comment:
If you call the component `Footer`, then this can't be called `Footer` - maybe `Wrapper` instead?